frontend/lib/src/components/elements/Expander/Expander.tsx

```diff
@@ -75,26 +72,30 @@ export const ExpanderIcon = (props: ExpanderIconProps): ReactElement => {
     )
   } else if (icon === "check") {
     return (
-      <StyledIcon
-        as={Check}
-        color={"inherit"}
+      <DynamicIcon
+        iconValue=":material/check:"
+        color="inherit"
         aria-hidden="true"
         data-testid="stExpanderIconCheck"
         {...iconProps}
       />
     )
   } else if (icon === "error") {
     return (
-      <StyledIcon
-        as={ErrorOutline}
-        color={"inherit"}
+      <DynamicIcon
+        iconValue=":material/error:"
+        color="inherit"
         aria-hidden="true"
         data-testid="stExpanderIconError"
         {...iconProps}
       />
     )
   }
 
```

Comment:
One consideration here could be to fully remove these two icons here and instead just set `:material/check:` or `:material/error:` in the backend for icon instead of `check` or `error`. Having that here feels a bit redundant since its basically doing the same as just using `:material/check:`. We would just need to remove or adapt the unit test that uses the testIDs here and check for the icon value instead.

Reply:
Ahh, why didn't I think of that. Really good point! I've set the material directive in the backend now.